### PR TITLE
PYIC-7677: Allow expired documents - content updates & welsh translations

### PR DESCRIPTION
**Pull request by AmritSidhu**
### What changed

A policy decision has been made to allow certain expired documents to be accepted as evidence when a user is proving their identity with One Login.

As such this means we only have to make small content updates on a few pages:
- /page-ipv-identity-document-types
- /pyi-post-office
- /no-photo-id-abandon-find-another-way 
- /prove-identity-another-type-photo-id/en?context=drivingLicence
- /prove-identity-another-type-photo-id/en?context=passport
- /no-photo-id-security-questions-find-another-way
- /no-photo-id-exit-find-another-way

### Issue tracking

- [PYIC-7677](https://govukverify.atlassian.net/browse/PYIC-7677)

### Environment variables or secrets

- [x] No environment variables or secrets were added or changed

[PYIC-7677]: https://govukverify.atlassian.net/browse/PYIC-7677?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: views/ipv/page/page-ipv-identity-document-types.njk [hunk 1]
```diff
@@ -1,54 +1,51 @@
...
+    <source srcset="/public/images/passport-icon.svg" />
+    <img width="174" height="121" loading="lazy" src="/public/images/passport-icon-1x.png" srcset="/public/images/passport-icon-2x.png 2x" alt="{{ 'pages.pageIpvIdentityDocumentTypes.content.imgAltText' | translate }}" />
+  </picture>
+
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+  <h2 class="govuk-heading-m">{{ 'pages.pageIpvIdentityDocumentTypes.content.subHeading4' | translate }}</h2>
+  <p class="govuk-body">{{ 'pages.pageIpvIdentityDocumentTypes.content.paragraph6' | translate }}</p>
+  <ul class="govuk-list  govuk-list--bullet">
+    {% for listItem in 'pages.pageIpvIdentityDocumentTypes.content.nonUkPassportRequirements' | translate({ returnObjects: true }) %}
```
Comment (blakeyp) [inline views/ipv/page/page-ipv-identity-document-types.njk]: I think nonUkPassportRequirements should be cardOrPermits here?

File: browser-tests/snapshot-tests/snapshot.spec.ts-snapshots/no-photo-id-exit-find-another-way-en-chromium-linux.jpeg
Comment (DanCorderIPV) [inline browser-tests/snapshot-tests/snapshot.spec.ts-snapshots/no-photo-id-exit-find-another-way-en-chromium-linux.jpeg:1]: The figma has a couple of things that don't seem to be in here:
- The text "You can use a:" inside the expanded area
- The word "only" at the end of the last paragraph in the expanded area

**Review by DanCorderIPV — CHANGES_REQUESTED** [review]

File: browser-tests/snapshot-tests/snapshot.spec.ts-snapshots/no-photo-id-exit-find-another-way-en-chromium-linux.jpeg
Comment (AmritSidhu) [reply browser-tests/snapshot-tests/snapshot.spec.ts-snapshots/no-photo-id-exit-find-another-way-en-chromium-linux.jpeg:1]: Good spots. 
In regards to the word 'only', i've based the translations off whats in the actual welsh translations spreadsheet which doesn't include that word. <- this was one of the discrepancies I mentioned i found during standup

Comment (AmritSidhu) [reply browser-tests/snapshot-tests/snapshot.spec.ts-snapshots/no-photo-id-exit-find-another-way-en-chromium-linux.jpeg:1]: I will double check again with UCD if this is required or not

Comment (DanCorderIPV) [reply browser-tests/snapshot-tests/snapshot.spec.ts-snapshots/no-photo-id-exit-find-another-way-en-chromium-linux.jpeg:1]: Thanks :)

**Review by DanCorderIPV — APPROVED** [review]

**Pull request merged**